.pre-commit-config.yaml

```diff
@@ -31,8 +31,3 @@ repos:
     rev: 25.1.0
     hooks:
       - id: black
-
```

Comment:
I had enough of this. False positive after false positive `every time` I merge main to my branch